networkd/networkd.ml

```diff
@@ -80,6 +80,8 @@ let doc = String.concat "\n" [
 	"This service looks after host network configuration, including setting up bridges and/or openvswitch instances, configuring IP addresses etc.";
 ]
 
+exception Notify_failed
```

Comment:
It is fix to one of the problems that arised after changes. I will remove the exception. It's not needed indeed